### PR TITLE
KOGITO-3252: management console configuration to use the process SVG …

### PR DESCRIPTION
**Pull request by nmirasch**
…addon

Jira: https://issues.redhat.com/browse/KOGITO-3252
Depends on https://github.com/kiegroup/kogito-runtimes/pull/846

### CODE REVIEW

File: management-console/pom.xml [hunk 1]
```diff
@@ -107,6 +112,22 @@
...
             </configuration>
           </execution>
+          <execution>
+            <id>copy-svg-resources</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/classes/META-INF/processSVG/</outputDirectory>
```
Comment (cristianonicolai) [inline management-console/pom.xml]: @nmirasch I though we were only going to point to a external folder in the management console. Not sure we should package the svgs in the jar, as the users wouldnt be able to do the same as we provide the generated app. Not sure if Im missing something.

Comment (nmirasch) [reply management-console/pom.xml]: yes, that was and currently is  the idea. I have revert this  resource copy into the jar. I have added some test and resources with  the needed svg at least for the kogito-travel-agency.

**Review by cristianonicolai — APPROVED** [review]

**Comment by nmirasch** [comment]
@cristianonicolai @jstastny-cz Just kept  the process-svg-addon dependency and added the native-imagen plugin

**Review by cristianonicolai — APPROVED** [review]

**Comment by nmirasch** [comment]
jenkins please retest this

File: management-console/pom.xml [hunk 2]
```diff
@@ -121,6 +125,21 @@
       </activation>
       <build>
         <plugins>
+          <plugin>
```
Comment (jstastny-cz) [inline management-console/pom.xml:129]: Do we actually need it? it's already defined in the common section in project / build, so just setting the property `<quarkus.package.type>native</quarkus.package.type>` below should be enough.

Comment (jstastny-cz) [reply management-console/pom.xml:129]: it's not enough :-)

**Review by jstastny-cz — APPROVED** [review]

**Pull request merged**